wormhole-connect/src/config/testnet.ts

```diff
@@ -147,6 +160,7 @@ export const TESTNET_TOKENS: { [key: string]: TokenConfig } = {
     decimals: 6,
     solDecimals: 6,
     suiDecimals: 8,
```

Comment:
Is this value `suiDecimals: 8` right for a USDC asset?

Reply:
Nope, this looks like n issue - it should be `6`